src/unity/python/README.md

```diff
@@ -1,132 +0,0 @@
-<img align="right" src="https://docs-assets.developer.apple.com/turicreate/turi-dog.svg" alt="Turi Create" width="100">
-
-# Turi Create 
-
-Turi Create simplifies the development of custom machine learning models. You
-don't have to be a machine learning expert to add recommendations, object
-detection, image classification, image similarity or activity classification to
-your app.
-
-* **Easy-to-use:** Focus on tasks instead of algorithms
-* **Visual:** Built-in, streaming visualizations to explore your data
-* **Flexible:** Supports text, images, audio, video and sensor data
-* **Fast and Scalable:** Work with large datasets on a single machine
-* **Ready To Deploy:** Export models to Core ML for use in iOS, macOS, watchOS, and tvOS apps
-
-Example: Image classifier with a few lines of code
```

Comment:
This section is not required in the pypi readme.

Reply:
I've changed the file according to what agreed here in the conversation. Please review again.